src/RotateView.js

```diff
@@ -22,15 +22,15 @@ type RotateViewProps = {
     componentDidMount() {
       this.controlId = uuid.v4();
       this.controlSelector = ".rotate-controls-" + this.controlId;
-      this.renderControls(this.props);
-    }
-    componentWillReceiveProps(nextProps: RotateViewProps) {
-      this.renderControls(nextProps);
     }
   
     componentWillUnmount() {
       this.clearControls();
     }
+
+    getDerivedStateFromProps(props) {
+      this.renderControls(props)
+    }
   
     renderControls(props: RotateViewProps) {
```

Comment:
I do not understand what side effect src/RotateView might have. Looks like it does not do anything.

I have fixed it anyway but if it is unused, we should delete the file.